src/org/jitsi/impl/neomedia/RTPConnectorOutputStream.java

```diff
@@ -663,6 +694,23 @@ private boolean write(RawPacket[] pkts)
         return success;
     }
 
+    /**
+     * @return the current output bitrate in bits per second.
+     */
+    public long getOutputBitrate()
```

Comment:
I think it makes it clearer. It could be measuring the bitrate of the stuff that is written to it, before transformations, possible drops of packets, and before it's actually sent (I am moving the call to update() that you mentioned).